tools/expotools/yarn.lock

```diff
@@ -4138,6 +4156,14 @@ chalk@^1.1.1, chalk@^1.1.3:
     strip-ansi "^3.0.0"
     supports-color "^2.0.0"
 
+chalk@^3.0.0:
```

Comment:
There's already `chalk@4.0.0` 😉

Reply:
I think it just got upgraded as a result of `inquirer` upgrade. It wasn't my intention to upgrade chalk at this time 😉